### ISSUE TITLE
Moving Forward

### ISSUE THREAD

**Issue by jpenzens** [UMCTM/sim-software-evaluation#5]
@vjd @dpastoor 

Lets keep this thread open to have a linear discussion on where to take this project.  Several parallel discussions can get confusing.

**Comment by dpastoor** [comment]
base on the comments #4 I would suggest taking the same tact with mrgsolve. Please try to chunk your commits a bit more (eg every step or set of logical steps) rather than 1 giant commit. You can always squash your commits down the line for a single review, but while working makes it easier to reference specific commits if you have questions

**Comment by jpenzens** [comment]
@vjd @dpastoor  Completely finished as of #7 .  Shall we move on to another simulator?

IMO mrgsolve is so fast and flexible i would be comfortable working on production-level code with it.

**Comment by dpastoor** [comment]
@jpenzens yes I will do a more thorough review of your mrgsolve stuff, however please take note of a couple of comments I made, especially regarding simulated distributions, and keep those in mind as you move forward.

Why don't you give a crack at RxODE. This one is a bit of a black box to us (we've only seen the paper, I copy-pasted their example once to run it but that is the extent of my use so we have no real clue how it is set up.
